<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clearance</class>
 <widget class="QWidget" name="clearance">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clearance</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background : qlineargradient(spread:pad, x1:0.011, y1:0.989, x2:1, y2:0, stop:0.0738636 rgba(255, 255, 255, 216), stop:1 rgba(170, 255, 255, 245))</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>176</width>
     <height>169</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="btn_Begin_Clearance">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 232, 232)</string>
      </property>
      <property name="text">
       <string>Start Clearance</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_Previous_Machine">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 232, 232)</string>
        </property>
        <property name="text">
         <string>Previous Machine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Next_Machine">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 232, 232)</string>
        </property>
        <property name="text">
         <string>Next Machine</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Manual">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 232, 232)</string>
      </property>
      <property name="text">
       <string>Manual Note Entry</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_Machine">
      <property name="styleSheet">
       <string notr="true">background-color: white</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Go_To">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 232, 232)</string>
      </property>
      <property name="text">
       <string>Go to machine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_End_Clearance">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 232, 232)</string>
      </property>
      <property name="text">
       <string>End Clearance</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="s_label1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string>Machine Count</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>251</width>
     <height>113</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLabel" name="l_total_pieces">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Total Amount:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="l_total_amount">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>$0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="l_grand_total">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>$0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Grand Total:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Total Pieces:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>185</width>
     <height>206</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="l_10count">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>$20:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="l_100count">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>$100:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="l_20count">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>$10:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="l_5count">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="l_50count">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>$5:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>$50:</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>251</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent</string>
      </property>
      <property name="text">
       <string>Machine status:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_machine_status">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent</string>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="text">
       <string>Machine No:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_current_machine_no">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background-color: transparent</string>
      </property>
      <property name="text">
       <string>Number</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="chk_Printer">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent</string>
   </property>
   <property name="text">
    <string>Printer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Clear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232)</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>211</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: transparent</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QListView" name="lv_recentlyMachine">
      <property name="styleSheet">
       <string notr="true">background-color: white</string>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QListView" name="lv_recentlyCounted">
      <property name="styleSheet">
       <string notr="true">background-color: white</string>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>s_label1</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>chk_Printer</zorder>
  <zorder>btn_Clear</zorder>
  <zorder>label_10</zorder>
  <zorder>label_9</zorder>
  <zorder>label_8</zorder>
  <zorder>l_total_amount</zorder>
  <zorder>l_total_pieces</zorder>
  <zorder>l_grand_total</zorder>
  <zorder>lv_recentlyCounted</zorder>
  <zorder>lv_recentlyMachine</zorder>
  <zorder>lv_recentlyMachine</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Begin_Clearance</sender>
   <signal>clicked()</signal>
   <receiver>clearance</receiver>
   <slot>OnBnClickedStartClearanceBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Next_Machine</sender>
   <signal>clicked()</signal>
   <receiver>clearance</receiver>
   <slot>OnBnClickedNextmachineBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Manual</sender>
   <signal>clicked()</signal>
   <receiver>clearance</receiver>
   <slot>OnBnClickedManualNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Go_To</sender>
   <signal>clicked()</signal>
   <receiver>clearance</receiver>
   <slot>OnBnClickedGoToMachineBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Go_To</sender>
   <signal>clicked()</signal>
   <receiver>edit_Machine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_End_Clearance</sender>
   <signal>clicked()</signal>
   <receiver>clearance</receiver>
   <slot>OnBnClickedSaveClearanceBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Clear</sender>
   <signal>clicked()</signal>
   <receiver>clearance</receiver>
   <slot>OnBnClickedClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>EndClicked(int,int)</signal>
  <slot>OnBnClickedSaveClearanceBtn()</slot>
  <slot>OnBnClickedStartClearanceBtn()</slot>
  <slot>OnBnClickedGoToMachineBtn()</slot>
  <slot>OnBnClickedNextmachineBtn()</slot>
  <slot>OnBnClickedManualNote()</slot>
  <slot>OnBnClickedClear()</slot>
 </slots>
</ui>
